### ISSUE TITLE
[ONNX] Keep functional ops as functions in dynamo exported onnx

### ISSUE THREAD

**Issue by nihui** [pytorch/pytorch#106601]
### 🚀 The feature, motivation and pitch

`nn.Module` may be represented as function in onnx
I was expecting the functional op to be expressed as such

Although most functional ops correspond to aten ops one-to-one, there are only a few parameter differences
But there are still some common functional ops that will be expressed as a combination of multiple aten ops, such as 
* F.normalize
* F.interpolate
* etc.

Representing it as a function can express the structure of the model more meaningfully, and is more friendly to the optimization of the backend

Otherwise downstream tools need to write rules and combine them back together, like https://github.com/Tencent/ncnn/blob/master/tools/pnnx/src/pass_level2/F_interpolate.cpp

**Comment by justinchuby** [comment]
Thanks for creating the issue. I checked with @BowenBao and learned that there may be current limitation in dynamo around capturing nn.functional ops. @BowenBao may have more info.

**Comment by MaanavD** [comment]
Hi @nihui ,

We’ve gone ahead and closed this issue because it is stale.
If you still believe this issue is relevant, please feel free to reopen the issue and we will triage it as necessary. 

Also pinging @justinchuby in case there's anything we need to follow up on.
